examples/Bunch/cmakeFlags

```diff
@@ -35,6 +35,16 @@ flags[2]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_INCLUDE_FIELDBACKGROUND
 flags[3]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_SINGLE_PARTICLE=true"
 flags[4]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DENABLE_SYNCHROTRON_PHOTONS=1"
 flags[5]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_FILTER_GAMMA=1"
+flags[6]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_RADWINDOW=radWindowFunctionTriangle"
```

Comment:
@ax3l I joined both test since they are independent from each other. (I am aware that this is not intended for a RT test - which would make no sense for this setup anyway)  

I will have a look at deactivating pusher, current and field2particle interpolation.